js/package.json

```diff
@@ -8,6 +8,7 @@
     "prettier:write": "prettier --write .",
     "type:check": "pnpm run -r type:check",
     "lint": "eslint . --ext .ts",
+    "test": "./node_modules/jest/bin/jest.js",
```

Comment:
this is not needed, we just need to remove the exit code on the semantic conventions. All tests can be run using `pnpm run -r test`